tests/functional/helpers/remote-repo.ts

```diff
@@ -14,7 +14,7 @@ export default class RemoteRepo extends Repo {
     await fs.promises.mkdir(this.getWorkingDirectoryPath(), {
       recursive: true,
     });
-    await this.runCommand('git', ['init', '--bare']);
+    await this.runCommand('git', ['init', '-b', 'main', '--bare']);
```

Comment:
@mcmire functional tests where failing due to default branch was set to master on pipeline although passes in Local that's why I did this change

Reply:
Makes sense!